beaker.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.5.4'
-  s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'json', '~> 1.4'
```

Comment:
I don't think it's bad. Kind of why I was just saying at least version 1, but really anything. I guess really just dropping the restriction altogether is 👍